ciao-deploy/deploy/nodes.go

```diff
@@ -160,7 +259,18 @@ func setupNode(ctx context.Context, anchorCertPath string, caCertPath string, ho
 		}
 	}()
 
-	err = InstallToolRemote(ctx, sshUser, hostname, "ciao-launcher", remoteCertPath, caCertPath)
+	err = InstallToolRemote(ctx, sshUser, hostname, unitFileConf{
+		Tool:       "ciao-launcher",
+		User:       ciaoUser,
+		CertPath:   remoteCertPath,
+		CACertPath: caCertPath,
+		Caps: []string{"CAP_NET_ADMIN", "CAP_NET_RAW", "CAP_DAC_OVERRIDE",
```

Comment:
Do we still need CAP_DAC_OVERRIDE? That's a pretty permissive capability...

Reply:
Afraid so.  I couldn't figure out how to get things to work without it.